distro_test.go

```diff
@@ -72,6 +72,44 @@ func isTestLinuxProcessAlive(d *wsl.Distro) bool {
 	return err == nil
 }
 
+func TestDefaultDistro(t *testing.T) {
+	want := newTestDistro(t, emptyRootFs)
+	out, err := exec.Command("wsl.exe", "--set-default", want.Name).CombinedOutput() //nolint: gosec
```

Comment:
can you add a comment on what we nolint?

Reply:
All the exec.Command //nolint are `G204: Subprocess launched with a potential tainted input or cmd arguments`.

I'll add the comment.

Reply:
you can add a rationale like "we control the argument in tests" or something like that.